pkg/trace/stats/statsraw.go

```diff
@@ -194,12 +187,15 @@ func (sb *RawBucket) HandleSpan(s *WeightedSpan, env string, aggregators []strin
 		panic("env should never be empty")
 	}
 
-	m := make(map[string]string)
+	m := []Tag{}
```

Comment:
```suggestion
	var m []Tag
```
We usually do it like this. Also is there a better name than `m`?